core/liblwm2m.h

```diff
@@ -643,6 +663,7 @@ typedef struct
     uint16_t                nextMID;
     lwm2m_transaction_t *   transactionList;
     void *                  userData;
+    lwm2m_block1_data_t *   block1DataList;
```

Comment:
about the cleanup and to do it on bootstrap if you put your block1_data structure in the 'server' and not on the 'lwm2m_context' the bootstrap process will cleanup the server after bootstraping to new servers.
I'm not sure it's possible :)